tenpy/linalg/np_conserved.py

```diff
@@ -4087,6 +4088,27 @@ def qr(a,
     return q, r
 
 
+def lq(a,
```

Comment:
I don't think I understand. I need some decomposition that leave the unitary matrix on the right. LQ seemed the best choice since it's cheaper than an SVD and just involves QR + transposes.

Reply:
I could use `polar` with `left=True`, but this works by doing an SVD, which I'm trying to avoid for efficiency reasons.